blocks/README.md

```diff
@@ -219,3 +219,35 @@ function edit( props ) {
 	} );
 }
 ```
+
+## Matchers
+
+Given the block's HTML content as a string, attribute matchers extract the block attributes values into an Object. The matchers are made available throught the `wp.blocks.query` variable as superset of [HPQ](http://github.com/aduth/hpq)'s API.
+
+`attr( selector: ?string, name: string ): Function`
+
+Generates a matcher function, returning an attribute by name if the attribute exists. If no selector is passed, returns attribute of the root element.
+
+`prop( selector: ?string, name: string ): Function`
+
+Generates a matcher function, returning an attribute by property if the attribute exists. If no selector is passed, returns property of the root element.
+
+`html( selector: ?string ): Function`
+
+Convenience for `prop( selector, 'innerHTML' )` .
```

Comment:
Maybe "Use if you want to get the raw html within the block as a single attribute. Convenience for `prop( selector, 'innerHTML' )`".